app/views/admin/suppliers/_layout.html.erb

```diff
@@ -1,17 +1,13 @@
 <h1 class="govuk-heading-l">Suppliers</h1>
 <div class="page-tab-container">
   <div class="page-tab govuk-!-margin-right-8 <%= "page-tab__selected" if page == "suppliers" %>">
-    <% if page == "suppliers" %>
-      All suppliers
-    <% else %>
-      <%= govuk_link_to "All suppliers", admin_suppliers_path %>
+    <%= govuk_link_to admin_suppliers_path do %>
+      All suppliers <%= "<span class='govuk-visually-hidden'>Current page</span>".html_safe if page == "suppliers" %>
```

Comment:
I think it's good enough solution for now.

For the future, if we want to keep using this navigation more often, I think we should make a more general component. Pity it's not a thing in govuk design system.

Reply:
Agreed